examples/contracts/aggregator_consumer/Makefile

```diff
@@ -1,12 +1,53 @@
-# TODO: these helper commands may be useful for the deploy script(s)
-#
-# ACCOUNT_NAME="test-acct"
-# 
-# create-account:
-# 	@sncast account create --name $(ACCOUNT_NAME)
-# 
-# deploy-account:
-# 	@sncast account deploy --name $(ACCOUNT_NAME)
-
-run-script:
-	@cd ./scripts && sncast script run $(NAME)
+# The starknet-devnet-rs container version - this version 
+# supports rpc v0.7 which is needed for snfoundry v0.20.1
+CONTAINER_VERSION="7743a089a33beb75d7012e4aa24745bee8ae0d71"
+
+# The name of the account that will be used for snfoundry
+# scripts + transactions
+ACCOUNT_NAME="test-acct"
+
+devnet:
+	@printf "\nStarting a local starknet devnet docker container:\n\n" \
+		&& bash ../../../ops/scripts/devnet-hardhat-down.sh \
```

Comment:
We already call cleanup on startup: https://github.com/smartcontractkit/chainlink-starknet/blob/38b10b2e272aed56d0538f39aee4b2e89411753a/ops/scripts/devnet-hardhat.sh#L11